test/schema/artist/index.js

```diff
@@ -18,11 +18,31 @@ describe('Artist type', () => {
       .onCall(0)
       .returns(Promise.resolve(42));
     Artist.__Rewire__('total', total);
+
+    const bio = sinon.stub();
+    bio
+      .onCall(0)
+      .returns(Promise.resolve(null));
+    Artist.__Rewire__('bio', bio);
+
+    const blurb = sinon.stub();
+    blurb
+      .onCall(0)
+      .returns(Promise.resolve(null));
+    Artist.__Rewire__('blurb', blurb);
```

Comment:
Instead of stubbing `bio` and `blurb`, I bet you can just add them to the stubbed Gravity response just above.